### PR TITLE
feat: add log-name option

### PR DESCRIPTION
**Pull request by Dup4**
related #1559

### CODE REVIEW

File: lib/auxlib.c [hunk 1]
```diff
@@ -300,6 +302,9 @@ void config_load(int *argc, char ***argv, GOptionEntry *app_entries, const char
 
 out:
 	// default common values, if not configured
+	if (rtpe_common_config_ptr->log_name == NULL)
+		rtpe_common_config_ptr->log_name = g_strdup("rtpengine");
+
```
Comment (rfuchs) [inline lib/auxlib.c:307]: Suggestion to make things more consistent towards the recording daemon: omit this part ...  (1/3)

File: daemon/main.c [hunk 2]
```diff
@@ -1023,7 +1023,7 @@ static void early_init(void) {
 }
 
 static void init_everything(void) {
-	log_init("rtpengine");
+	log_init(rtpe_common_config_ptr->log_name);
```
Comment (rfuchs) [inline daemon/main.c:1026]: ... leave this one unchanged ... (2/3)

**Review by rfuchs — COMMENTED** [review]
... and instead put the logic into `log_init()`: if `log_name` is set, use it, or otherwise use the default from the function argument `handle` (3/3)

Does this make sense? Just a suggestion anyway... If you prefer to leave it as it is, I will merge it.

**Comment by Dup4** [comment]
> ... and instead put the logic into `log_init()`: if `log_name` is set, use it, or otherwise use the default from the function argument `handle` (3/3)
> 
> Does this make sense? Just a suggestion anyway... If you prefer to leave it as it is, I will merge it.

I prefer to leave it as it is.

Because I prefer to keep a function clean, rather than coupled with a global configuration.

**Pull request closed without merging**